### ISSUE TITLE
Tabulacion incorrecta.

### ISSUE THREAD

**Issue by willy5360** [Heylga/Photofeed#1]
https://github.com/Heylga/Photofeed/blob/81d4fdbf0b19982f2a1fb5eb596222792f9cbd54/index.html#L7

El tag header no esta bien tabulado

**Comment by NaincyKumariKnoldus** [comment]
Hey @willy5360 Can I do this please?

**Comment by NaincyKumariKnoldus** [comment]
Hola @willy5360 , había planteado un PR para esto. Por favor, revísalo.

**Comment by willy5360** [comment]
Correcto. Cuando hayas terminando de corregir todos los apartados, ya recien me envias una PR. y vuelves a enviar este proyecto por breathcode.

**Comment by NaincyKumariKnoldus** [comment]
Hola @willy5360 , He planteado relaciones públicas para esto. Puedes fusionar esto.

**Comment by willy5360** [comment]
No puedo fusinarlo porque yo no tengo permisos sobre tu repo.

**Comment by NaincyKumariKnoldus** [comment]
He planteado relaciones públicas para su repositorio. Puedes hacerlo.
